usersexercise/src/main/java/com/Model/Response/UserResponse.java

```diff
@@ -0,0 +1,44 @@
+package com.Model.Response;
+
+public class UserResponse {
+
+    private Long id;
+    private String first_name;
+    private String last_name;
+    private String email;
+
+    public UserResponse(){
+        super();
+    }
+
+    public UserResponse(String first_name, String last_name, String email) {
+        super();
+        this.first_name = first_name;
+        this.last_name = last_name;
+        this.email = email;
+    }
+
```

Comment:
You need getters and setters for id in all your models that have id. I appreciate you taking to heart to think about your fields and only include the functionality if you need it, but in this case you do need it. Your id was coming back null in your response because `BeanUtils.copyProperties` is really just a shortcut to use getters and setters to copy the values of matching properties. Without the getters and setters the information isn't transferred.